plugins/plugin-chart-echarts/src/Gauge/controlPanel.tsx

```diff
@@ -0,0 +1,267 @@
+              type: 'CheckboxControl',
+              label: t('Show axis line ticks'),
+              renderTrigger: true,
+              default: showAxisTick,
+              description: t('Show axis line ticks'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'start_angle',
+            config: {
+              type: 'TextControl',
+              label: t('Start angle'),
+              renderTrigger: true,
+              default: startAngle,
+              description: t('Angle at which to start progress axis'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'end_angle',
+            config: {
+              type: 'TextControl',
+              label: t('End angle'),
+              renderTrigger: true,
+              default: endAngle,
+              description: t('Angle at which to end progress axis'),
+            },
+          },
+        ],
+        [<h1 className="section-header">{t('Intervals')}</h1>],
+        ['color_scheme'],
+        [
+          {
+            name: 'intervals',
+            config: {
+              type: 'TextControl',
+              label: t('Interval bounds'),
+              description: t(
+                'Comma-separated interval bounds, e.g. 2,4,5 for intervals 0-2, 2-4 and 4-5. Last number should match the value provided for MAX.',
+              ),
+              default: intervals,
+            },
+          },
+        ],
+        [
+          {
+            name: 'interval_color_indices',
+            config: {
+              type: 'TextControl',
+              label: t('Interval colors'),
+              description: t(
+                'Comma-separated color picks for the intervals, e.g. 1,2,4. Integers denote colors from the chosen color scheme and are 1-indexed. Length must be matching that of interval bounds.',
+              ),
+              default: intervalColorIndices,
+            },
+          },
+        ],
+        [<h1 className="section-header">{t('Value')}</h1>],
+        [
+          {
+            name: 'value_name',
+            config: {
+              type: 'TextControl',
+              label: t('Value name'),
+              description: t('Value name'),
+              renderTrigger: true,
+              default: valueName,
+            },
+          },
+        ],
+        [
+          {
+            name: 'value_formatter',
+            config: {
+              type: 'TextControl',
+              label: t('Value format'),
+              renderTrigger: true,
+              default: valueFormatter,
+              description: t('Value format'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'value_animation',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Value animation'),
+              renderTrigger: true,
+              default: valueAnimation,
+              description: t('Animate the value or just display it'),
```

Comment:
I am not sure if this is required.
Does it only control value animation or arrow animation as well?
If it only for value, then I think we should give the single checkbox to enable/disable both arrow and value animation.

Reply:
This controls only value animation but there is actually a setting for both so I will change this.

Reply:
It does not seem to work great inside Superset so I will just drop it.